Formula/symlinks.rb

```diff
@@ -0,0 +1,26 @@
+class Symlinks < Formula
+  desc "Symbolic link maintenance utility"
+  homepage "https://github.com/brandt/symlinks"
+  url "https://github.com/brandt/symlinks/archive/v1.4.3.tar.gz"
+  sha256 "27105b2898f28fd53d52cb6fa77da1c1f3b38e6a0fc2a66bf8a25cd546cb30b2"
+  license "MIT"
+
+  def install
+    system ENV.cc, "-Wall", "-Wstrict-prototypes", "-O2", "-o", "symlinks", "symlinks.c"
```

Comment:
Is there no makefile or something?

Reply:
It has a single-line one but it explicitly calls `gcc` which I was trying to avoid, in line with Homebrew's "should use current clang" requirement.

Reply:
Then this should work:
```
system "make", "CC=#{ENV.cc}", "symlinks"
```

Reply:
Ah, TIL about fixing that in make, thanks. Sorted.